dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/AggregateMetricsCollector.java

```diff
@@ -78,8 +85,16 @@ public AggregateMetricsCollector(ApplicationModel applicationModel) {
             if (optional.isPresent()) {
                 registerListener();
                 AggregationConfig aggregation = optional.get().getAggregation();
-                this.bucketNum = aggregation.getBucketNum() == null ? DEFAULT_BUCKET_NUM : aggregation.getBucketNum();
-                this.timeWindowSeconds = aggregation.getTimeWindowSeconds() == null ? DEFAULT_TIME_WINDOW_SECONDS : aggregation.getTimeWindowSeconds();
+                this.bucketNum = aggregation.getBucketNum() == null ?
+                    DEFAULT_BUCKET_NUM : aggregation.getBucketNum();
+                this.timeWindowSeconds = aggregation.getTimeWindowSeconds() == null ?
+                    DEFAULT_TIME_WINDOW_SECONDS : aggregation.getTimeWindowSeconds();
+                this.qpsTimeWindowMillSeconds = aggregation.getQpsTimeWindowMillSeconds() == null ?
+                    DEFAULT_QPS_TIME_WINDOW_MILL_SECONDS : aggregation.getQpsTimeWindowMillSeconds();
+                this.enableQps = aggregation.getEnableQps() == null || aggregation.getEnableQps();
+                this.enableRtPxx = aggregation.getEnableRtPxx() == null || aggregation.getEnableRtPxx();
+                this.enableRt = aggregation.getEnableRt() == null || aggregation.getEnableRt();
+                this.enableRequest = aggregation.getEnableRequest() == null || aggregation.getEnableRequest();
             }
```

Comment:
Maybe it could be written more cohesive
public boolean getEnableQps() {
        return Optional.ofNullable(enableQps).orElse(true);
    }

Reply:
done